From 705a9ad229de49cd9cce226a2025716902455663 Mon Sep 17 00:00:00 2001
From: Philipp Huebner <debalance@debian.org>
Date: Wed, 20 Jan 2016 08:13:21 -0500
Subject: [PATCH] correct include

This part of the code was moved into it's own project and was packaged
separately by me. To make the build process work, this small fix is
necessary.
---
 src/mod_sip.erl           | 2 +-
 src/mod_sip_proxy.erl     | 2 +-
 src/mod_sip_registrar.erl | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/mod_sip.erl b/src/mod_sip.erl
index c8c9533..7f188a0 100644
--- a/src/mod_sip.erl
+++ b/src/mod_sip.erl
@@ -38,7 +38,7 @@
 
 -include("ejabberd.hrl").
 -include("logger.hrl").
--include_lib("esip/include/esip.hrl").
+-include_lib("p1_sip/include/esip.hrl").
 
 %%%===================================================================
 %%% API
diff --git a/src/mod_sip_proxy.erl b/src/mod_sip_proxy.erl
index bf297bc..d8b65cd 100644
--- a/src/mod_sip_proxy.erl
+++ b/src/mod_sip_proxy.erl
@@ -40,7 +40,7 @@
 
 -include("ejabberd.hrl").
 -include("logger.hrl").
--include_lib("esip/include/esip.hrl").
+-include_lib("p1_sip/include/esip.hrl").
 
 -define(SIGN_LIFETIME, 300). %% in seconds.
 
diff --git a/src/mod_sip_registrar.erl b/src/mod_sip_registrar.erl
index 44091ed..4d2ec82 100644
--- a/src/mod_sip_registrar.erl
+++ b/src/mod_sip_registrar.erl
@@ -37,7 +37,7 @@
 
 -include("ejabberd.hrl").
 -include("logger.hrl").
--include_lib("esip/include/esip.hrl").
+-include_lib("p1_sip/include/esip.hrl").
 
 -define(CALL_TIMEOUT, timer:seconds(30)).
 -define(DEFAULT_EXPIRES, 3600).
-- 
2.5.0

